app/apollo/resolvers/channel.js

```diff
@@ -211,15 +207,14 @@ const channelResolvers = {
       await validAuth(me, org_id, ACTIONS.CREATE, TYPES.CHANNEL, queryName, context);
 
       // Create the channel object to be saved.
-      const uuid = UUID();
       const kubeOwnerId = await models.User.getKubeOwnerId(context);
       const newChannelObj = {
         _id: UUID(),
-        uuid,
+        uuid: UUID(),
```

Comment:
Do we want  `_id != uuid` ?

Reply:
It could, though this would be a change from previous behavior.  The separate values for `uuid` and `_id` _are_ confusing, but making them use the same value might make future editors believe they can switch between using the two for queries for example, which would cause problems with old data created when they were different.

Reply:
I see they are diff for other objects (in other code), but this code previously explicitly kept them the same.